ios/tools/prepRelease.sh

```diff
@@ -89,9 +89,15 @@ cd $WORK_DIR
 KEYMANAPP_IPA="build/Build/Products/Release-iphoneos/Keyman.ipa"
 KEYMANAPP_IPA_DST="keyman-ios-${BUILD_NUMBER}.ipa"
 
-echo "Copying Keyman IPA ${KEYMANAPP_IPA} => ${UPLOAD_DIR}/${KEYMANAPP_IPA/DST}..."
+echo "Copying Keyman IPA ${KEYMANAPP_IPA} => ${UPLOAD_DIR}/${KEYMANAPP_IPA_DST}..."
 cp "${KEYMANAPP_IPA}" "${WORK_DIR}/${UPLOAD_DIR}/${KEYMANAPP_IPA_DST}"
 
+KEYMANAPP_SIM_IPA="build/Build/Products/Release-iphonesimulator/Keyman.ipa"
+KEYMANAPP_SIM_IPA_DST="keyman-ios-simulator-${BUILD_NUMBER}.ipa"
+
+echo "Copying Keyman simulator artifact ${KEYMANAPP_SIM_IPA} => ${UPLOAD_DIR}/${KEYMANAPP_SIM_IPA_DST}..."
+cp "${KEYMANAPP_IPA}" "${WORK_DIR}/${UPLOAD_DIR}/${KEYMANAPP_SIM_IPA_DST}"
```

Comment:
Why isn't this
`cp "${KEYMANAPP_SIM_IPA}" ...` ?

Reply:
```suggestion
cp "${KEYMANAPP_SIM_IPA}" "${WORK_DIR}/${UPLOAD_DIR}/${KEYMANAPP_SIM_IPA_DST}"
```